index.html

```diff
@@ -1,173 +1,352 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <title>
-            This is the title of the page
-        </title>
+        <title>Hf - Balog Martin</title>
+
+        <!-- GoogleFont-->
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap" rel="stylesheet">
+
+        <link rel="stylesheet" href="style.css" >
     </head>
     <body>
-        <a id="anchorID1" href="#anchorID2">Ugrás az oldal aljára</a>
-        <h1>Cimsor 1 - Paragrafus, cimsor, br-tag es hr-tag gyakorlas</h1>
-        <h2>Cimsor 2 - Lorem ipsum rövid</h2>
-            <p>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
-                Nullam vestibulum orci a sapien imperdiet maximus. 
-                Suspendisse vel ex eget mauris efficitur finibus eu a enim. 
-                Aenean tellus neque, consectetur sit amet lacus a, hendrerit ornare urna. 
-                Fusce venenatis lorem vitae augue suscipit, eu tempor massa vulputate. 
-                Praesent iaculis velit libero, vel faucibus sem pulvinar vel. 
-                Curabitur viverra nisl dui, a venenatis felis pellentesque id. 
-                Donec non lacus diam.
-            </p>
+        <div class="div_Navigation">
```

Comment:
Nem praktikus beleírni a classnevekbe a taget, amire alkalmazod. Egyrészt ezt meg tudod oldani szelektorral (`div.navigation`), másrészt, ha kicseréled a taget divről navra monjuk, akkor vagy hazudni fog a class, vagy módosítanod kell a class nevét is a taggel együtt (és mi indokolná a classnév változtatását, amikor struktúrát módosítasz?).
Illetve látok egy aláhúzást, mint elválasztót ebben a classnévben, de a többiben nem, így nem konzisztens a nevezék.

Reply:
Szia @frontendes-csabi, modositottam (commit 34e3b6a). Igy viszont most van egy `div.Navigation` es `a.Navigation` class. Ez igy okes? Nem problema, hogy mindkettöben a "Navigation" szot hasznalom?

Reply:
A dolog trükkje az, hogy a classnevek (ha nem utility classokat használsz) jelentést is hordoznak magukkal. Ha egy másik fejlesztő olvassa a HTML forrását és meglátja a `Navigation` classnevet több helyen, akkor egész biztos, hogy azonos jelentést is fog nekik tulajdonítani, ezáltal azonos stílusra is számít. Márpedig, ha jól olvasom a kódot, itt a `div` és az `a` elemeknek más szerepük van, ez extra gondolkodást fog eredményezni a részéről.

De amúgy látom, hogy már módosítottad, szerintem te is érezted ugyanezt :)

Reply:
Utóbibban sokat fog segíteni a JSDoc vagy a TypeScript, attól függ, melyik passzol a projekthez.